.eslintignore

```diff
@@ -0,0 +1 @@
+public
```

Comment:
Shouldn't this be `public/addon-build` and `public/build`? We also have other files in there (but ok, most of them don't require linting since they are images :) ).